src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const divElement = document.createElement('div');
+  const heading = document.createElement('h2');
+  const paragraph = document.createElement('p');
+
+  divElement.classList.add('notification', type);
+  divElement.style.top = `${posTop}px`;
+  divElement.style.right = `${posRight}px`;
+
+  heading.classList.add('title');
+  heading.textContent = title;
+  paragraph.textContent = description;
+
+  divElement.appendChild(heading);
+  divElement.appendChild(paragraph);
+
+  document.body.appendChild(divElement);
+
+  setTimeout(() => {
+    divElement.style.display = 'none';
+  }, 2000);
```

Comment:
Setting the display property to 'none' will not remove the element from the DOM, it will just hide it. Over time, if many notifications are created, the DOM could become bloated with hidden elements, which could impact performance. It would be better to actually remove the element from the DOM after it has been displayed. You can use `divElement.remove()` to achieve this.